packages/react/package.json

```diff
@@ -214,6 +216,7 @@
     "storybook-addon-turbo-build": "2.0.1",
     "styled-components": "5.3.11",
     "terser": "5.27.0",
+    "ts-node": "10.9.2",
```

Comment:
We have the `tsx` utility for running ts files if that helps! Sadly we had to move off of `ts-node` for some other scripts, just wanted to share that utility in case it wasn't obvious.